src/verifyEmail.js

```diff
@@ -0,0 +1,27 @@
+function verifyEmail(appId, serverURL) {
```

Comment:
I actually feel strongly here - we should be creating a MailController here, since we need a databaseAdapter here, which is state of a controller, which is not injected right now - but we are definitely moving that direction.